### DISCUSSION TITLE
SGBD supportés

### DISCUSSION THREAD

**Discussion by alain-mercier** [dbarzin/deming#37] [General]
Bonjour, je n'ai pas encore testé le logiciel mais il me semble très intéressant. Merci pour ce développement.
Quel est le problème avec le support de MariaDB ? Est il prévu une évolution pour permettre son utilisation à la place de MySQL ?

**Comment by dbarzin** [comment]
Cela n'a pas été testé mais c'est supporté par le framework Laravel qui est utilisé :  https://laravel.com/docs/10.x/database
